lesson 4

pvvchip opened the pull request:
Не знаю как:
- автоматическая прокрутка JscrollPane в конец
- после нажатия sendButton фокус активности перекинуть на chatTextField

pvvchip committed:
lesson 4
Незнаю как:
- автоматическая прокрутка JscrollPane в конец
- после нажатия sendButton фокус активности перекинуть на chatTextField

1000001rtem reviewed (commented):
в messageSend добавь: chatTextArea.setCaretPosition(chatTextArea.getDocument().getLength());
chatTextField.setCaretPosition(chatTextField.getDocument().getLength());

pvvchip committed:
lesson 4
Незнаю как:
- после нажатия sendButton фокус активности перекинуть на chatTextField

Победил
- автоматическая прокрутка JscrollPane в конец

pvvchip committed:
lesson 4

Победил
- автоматическая прокрутка JscrollPane в конец
- после нажатия sendButton фокус активности перекинуть на chatTextField

batiaev reviewed (commented):
по дз все хорошо
